README.md

```diff
@@ -43,7 +43,7 @@ Benchmark performance tests run nightly against the API (running in Docker Compo
 
 Full details of how the tests work can be found in [the Artillery readme](https://github.com/ministryofjustice/send-legal-mail-to-prisons-api/tree/main/artillery).
 
-You may see some alerts in the `#farsight-alerts` Slack channel about "SLM Performance errors". Details about how test results are recorded and alerted upon can also be found in [the Artillery readme](https://github.com/ministryofjustice/send-legal-mail-to-prisons-api/tree/main/artillery#benchmark-performance-tests).
+You may see some alerts in the `#send-legal-mail-alerts` Slack channel about "SLM Performance errors". Details about how test results are recorded and alerted upon can also be found in [the Artillery readme](https://github.com/ministryofjustice/send-legal-mail-to-prisons-api/tree/main/artillery#benchmark-performance-tests).
```

Comment:
These alerts are configured in Azure to [send an email to the farsight-alerts Slack channel ](https://portal.azure.com/#view/Microsoft_Azure_Monitoring/UpdateLogSearchV2AlertRuleViewModel/alertId/%2Fsubscriptions%2Fc27cfedb-f5e9-45e6-9642-0fad1a5c94e7%2FresourceGroups%2Fnomisapi-t3-rg%2Fproviders%2Fmicrosoft.insights%2Fscheduledqueryrules%2FSLM%20-%20Performance%20Benchmark%20dropped%20by%2020%20percent)- I think it will need a [new Action Group](https://portal.azure.com/#view/Microsoft_Azure_Monitoring/NotificationGroupsBlade/queryInputs~/%7B%22subscriptionIds%22%3A%5B%22c27cfedb-f5e9-45e6-9642-0fad1a5c94e7%22%5D%2C%22resourceGroups%22%3A%5B%22%2Fsubscriptions%2Fc27cfedb-f5e9-45e6-9642-0fad1a5c94e7%2FresourceGroups%2Fnomisapi-t3-rg%22%5D%7D) to send the alerts to the new channel.

Reply:
To find the email address of a Slack channel open its properties in Slack and look at the `Integrations` tab. Unless I've missed something and you've found a better way than sending emails! 😁

Reply:
Thanks, Mike! Yes, I'd found that in Azure but couldn't see how it was configured or how to change it as I don't have permissions. I've requested these so am just waiting on that (so the documentation for this specific bit has got slightly ahead of reality...).

Useful tip on the Slack/email integration 👍 - I'll use that when I get access. (For changing over Pingdom and AlertManager I created Slack webhooks...)